lib/app/views/server_state.erb

```diff
@@ -0,0 +1,19 @@
+<div id="server-state">
+  <h4>Valuesets Used in Inferno:</h4>
+  <table class="table table-striped">
+    <thead>
+      <tr>
+        <th>Valueset URL</th>
+        <th>Code count</th>
```

Comment:
Should this be `Concept Count`.

That seems to be how they are normally referred to:

e.g.
https://www.hl7.org/fhir/valueset-patient-contactrelationship.html#expansion
https://www.hl7.org/fhir/us/core/ValueSet-us-core-careteam-provider-roles.html#expansion